hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -37,12 +37,64 @@ def filter_params(complete, success, attributes):
 
 class Job:
     @staticmethod
-    def exit_code(job_status):
-        if 'exit_code' not in job_status or job_status['exit_code'] is None:
+    def _get_error(job_status, task):
+        status = job_status.get('status')
+        if not status:
+            return None
+
+        # don't return status error
+
+        container_statuses = status.get('container_statuses')
+        if not container_statuses:
+            return None
+
+        container_status = container_statuses.get(task)
+        if not container_status:
+            return None
+
+        error = container_status.get('error')
+        if error:
+            return error
+
+        dcontainer_status = container_status.get('container_status')
```

Comment:
This took me awhile to realize it was the docker_container_status. It might be better to spell out docker.